tests/test_lr_finder.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+from torch_lr_finder import LRFinder
+
+from . import task as mod_task
+
+
+def collect_task_classes():
+    from . import task as mod
+    from .task import BaseTask
```

Comment:
Any reason for the local imports? I think these can be deleted and in the function replace `task` with `mod_task` and `BaseTask` with `mod_task.BaseTask` (except for the string 'BaseTask' of course)

Reply:
You are right. These lines were for keeping global namespace clean while I intended to make all test cases able to be re-run automatically with various tasks. But since I found it not necessary to do that for some cases (e.g. `test_reset`) and add `from . import task as mod_task` at the top of script, it's fine to remove these lines.
Thanks for pointing this out. :)